concepts/bot_builder/code_webhook.md

```diff
@@ -1,29 +1,42 @@
 ---
 layout: concept
-title: Code and webhook
+title: Custom code and webhook
 permalink: /concepts/code-and-webhook
 ---
 
 
-If you want to extend Bot Builder with custom code, you can call your API with webhook actions.
+At many points in your conversation, you most likely want to retrieve business information or connect to an external system to perform actions. You can do this through **webhooks**.
+A webhook is a simple HTTP call to your backend. To configure your HTTP call, click Webhook action in the Bot Builder.
 
-![Recast.AI - Webhook url](//cdn.recast.ai/man/recast-ai-webhook-view.png)
+[IMAGE HERE]
 
-You must provide a full url or route (starting with a '/') to be called by Bot Builder, allowing you to run code or API calls.
+You can provide the full URL or route (starting with a '/') to be called by the Bot Builder. If you provide a route, it will be added to your bot's base URL (configurable in your bot's settings).

```

Comment:
```
If you provide a route, it will be added to your bot's base URL (configurable in your bot's settings).
```
Maybe this instead ? :
```
If you provide a route, the 'Bot webhook base URL' (configurable in your bot's settings) will be prepended to it.
```

Reply:
Yes ;)